src/components/PatientRecord/PatientRecord.tsx

```diff
@@ -14,29 +14,123 @@ import {
   ReportsVisualizer
 } from 'fhir-visualizers';
 import { DomainResource } from 'fhir-objects';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faNotesMedical,
+  faChevronLeft,
+  faChevronUp,
+  faChevronDown
+} from '@fortawesome/free-solid-svg-icons';
+import styles from './PatientRecord.module.scss';
+import { IconProp } from '@fortawesome/fontawesome-svg-core';
+
 const getResourceByType = (
   patientRecord: ReadonlyArray<DomainResource>,
   resourceType: string
 ): ReadonlyArray<object> => {
   return patientRecord.filter(resource => resource.resourceType === resourceType);
 };
 
-const PatientRecord: FC = () => {
+interface PatientRecordProps {
+  headerElement: React.RefObject<HTMLDivElement>;
```

Comment:
would be nice to import this the same way for other React bindings.
i.e. on line 1
`import React, { FC, useState, useEffect, useRef, RefObject } from 'react';`

And then here:
```suggestion
  headerElement: RefObject<HTMLDivElement>;
```

Reply:
Updated import